applications/luci-app-watchcat/htdocs/luci-static/resources/view/watchcat.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+'require view';
+'require form';
+'require tools.widgets as widgets';
+
+return view.extend({
+	render: function () {
+		var m, s, o;
+
+		m = new form.Map('watchcat', 
+			_('Watchcat'), 
+			_("Here you can set up several checks and actions to take in the event that a host becomes unreachable. \
+			Click the <b>Add</b> button at the bottom to set up more than one action."));
+
+		s = m.section(form.TypedSection, 'watchcat', _('Watchcat'), _('These rules will govern how this device reacts to network events.'));
+		s.anonymous = true;
+		s.addremove = true;
+
+		s.tab('general', _('General Settings'));
+
+		o = s.taboption('general', form.ListValue, 'mode',
+			_('Mode'),
+			_("Ping Reboot: Reboot this device if a ping to a specified host fails for a specified duration of time. <br> \
```

Comment:
Please change all `<br>` to `<br />`, otherwise it fails to render with XHTML themes.